execution/sym_region_frag_machine.ml

```diff
@@ -775,22 +792,24 @@ struct
       in
       let v = self#eval_int_exp_simplify exp in
 	try
-	  (Some 0, to_concrete v)
+	  (Some 0, to_concrete v, false, V.Constant(V.Int(V.REG_64, 0L)), 0L)
 	with NotConcrete _ ->
 	  let e = to_symbolic v in
 	  let eip = self#get_eip in
-	    if !opt_trace_sym_addrs then
+	    if !opt_trace_sym_addrs then (
```

Comment:
Do you find that adding the extra set of parentheses and semicolon in this case makes the code easier to read? I consider the extra grouping helpful when the body of the if is complex or it's nested within other if statements, but I usually skip it when the if is just guarding a tracing printf.